ingest/scripts/group_segments.py

```diff
@@ -232,8 +232,13 @@ def main(
 
         row["submissionId"] = joint_key
 
+        # Hash of all metadata fields should be the same if
+        # 1. field is not in keys_to_keep and
+        # 2. field is in keys_to_keep but is ""
+        filtered_record = {k: v for k, v in row.items() if v}
```

Comment:
We should probably coerce values to string here, so that a 0 integer still gets hashed